src/renderers/WebGLRenderer.js

```diff
@@ -2222,7 +2222,13 @@ function WebGLRenderer( parameters ) {
 
 			uniforms.normalMap.value = material.normalMap;
 			uniforms.normalScale.value.copy( material.normalScale );
-			if ( material.side === BackSide ) uniforms.normalScale.value.negate();
+			if ( material.side === BackSide ) {
+
+				var normalScaleZ = uniforms.normalScale.value.z;
```

Comment:
Maybe this code is more clear?

```js
uniforms.normalScale.value.x = - uniforms.normalScale.value.x;
uniforms.normalScale.value.y = - uniforms.normalScale.value.y;
```

Reply:
Yeah that is way more readable. Should have done that instead. Thank you for your improvement.

Reply:
Can you please update the PR with @mrdoob suggestion? 😉